content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md

```diff
@@ -0,0 +1,47 @@
+---
+date: '2023-07-04'
+title: "Getting the UTXO Library"
+project: corda
+version: 'Corda 5.0 Beta 4'
+menu:
+  corda5:
+    identifier: corda5-utxo-ledger-getting-utxo-library
+    parent: corda5-utxo-advanced-ledger-extensions
+    weight: 4490
+section_menu: corda5
+---
+
+# Getting the UTXO Library
+
+The Corda 5 Advanced UTXO Ledger Extensions library is an external library and not part of Corda 5. It needs to be sourced
+from Maven Central Repository or similar.
+
+1. Ensure that the repositories block of the project has `mavenCentral()` declared in the following way:
+
```

Comment:
Should the code blocks all be indented by three spaces?

Reply:
Why 3 spaces ... either 2, 4, or 8 right ?

Reply:
indented